src/utils/streams.js

```diff
@@ -36,7 +39,7 @@ export class HasherPassThrough extends Transform {
   // This method returns the computed sha1 from the data that passed through the
   // transform.
   sha1(): string {
-    this.log.verbose('sha1()');
+    this.log.trace('sha1()');
```

Comment:
Using `trace` generates a stack automatically which is nice when debugging. To see trace logs we need to run tests with `LOG_LEVEL=trace`, they don't show up when the log level is "verbose", so it's also more silent.